staging/src/k8s.io/kube-scheduler/config/v1alpha2/types_pluginargs.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	v1 "k8s.io/kube-scheduler/config/v1"
+)
+
+// InterPodAffinityArgs holds arguments used to configure the InterPodAffinity plugin.
+type InterPodAffinityArgs struct {
```

Comment:
For this and all `${PLUGIN}Args`.

- add TypeMeta field (see KubeSchedulerConfig type)
- add a deepcopy-gen comment (see KubeSchedulerConfig type)
- register the type in `staging/src/k8s.io/kube-scheduler/config/v1alpha2/register.go`

Reply:
Done